frontend/src/components/pages/Scheduling/VolunteerInformation.tsx

```diff
@@ -0,0 +1,32 @@
+import { Button, Container, HStack, Text } from "@chakra-ui/react";
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+import { SchedulingStepProps } from "./types";
+
+const VolunteerInformation = ({
+  formValues,
+  setForm,
+  navigation,
+}: SchedulingStepProps) => {
+  const { previous, next } = navigation;
+  const history = useHistory();
```

Comment:
Do we need this here?

Reply:
hmm yeah don't think we need this, we can remove it when working on this page